src/Model/RequestManager.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Created by WCS.
+ * User: Kevin
+ * Date : 28/10/2020
+ */
+
+namespace App\Model;
+
+class RequestManager extends AbstractManager
+{
+    const TABLE = 'request';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    public function selectFirsts(): array
+    {
+        return $this->pdo->query('SELECT * FROM ' . self::TABLE . ' JOIN ' . UserManager::TABLE .
```

Comment:
You probably should check errors because fetchAll() can return false, which is not an array (the type you declare the function must return)

Reply:
Ok, checked.